server/lib/src/constants/groups.rs

```diff
@@ -410,7 +410,6 @@ lazy_static! {
     };
 
 
-    /// Builtin IDM Group for allowing migrations of service accounts into persons
```

Comment:
It'd be really nice if there was some way of automagically making this comment match the description so when one looks at it in the docs/an IDE, you can tell what it does without having to dig deeper 😞